assembl/static2/js/app/reducers/adminReducer/timeline.js

```diff
@@ -59,7 +82,9 @@ export const phasesById: PhasesByIdReducer = (state: PhasesByIdState = Map(), ac
         start: start,
         end: end,
         isThematicsTable: isThematicsTable || false, // default to false until we have the interface to set a thematicstable
-        id: id
+        id: id,
+        order: order,
+        hasConflictingDates: getHasConflictingDates(action.phases, id, start, end) || false
```

Comment:
why do you need `hasConflictingDates` in the store? I think it can be managed by the component.

Reply:
Because 2 different components need it at the moment and possibly more in the future